src/backend/distributed/deparser/deparse_database_stmts.c

```diff
@@ -85,6 +86,14 @@ AppendGrantOnDatabaseStmt(StringInfo buf, GrantStmt *stmt)
 }
 
 
+static void
+AppendDefElemIsTemplate(StringInfo buf, DefElem *def)
+{
+	appendStringInfo(buf, " %s  %s", quote_identifier(def->defname),
+					 quote_literal_cstr(strVal(def->arg)));
+}
+
+
```

Comment:
This seems unrelated to this PR and should be done in #7172 instead.

Reply:
Fixed